docs/examples/1d_hist/1d_profile.py

```diff
@@ -0,0 +1,31 @@
+"""
+1d Profile
+==========
+
+Plot a 1d mean histogram (profile plot).
+"""
+
+###
+import boost_histogram as bh
+from plothist import plot_error_hist
+import matplotlib.pyplot as plt
+
+fig, ax = plt.subplots()
```

Comment:
put this just before `plot_error_hist(h, ax=ax)`, to clearly separate the histograming from the plotting.